examples/counter/reducers/index.js

```diff
@@ -1 +1,8 @@
-export { default as counter } from './counter';
+import { combineReducers } from 'redux';
+import counter from './counter';
+
+const rootReducer = combineReducers({
+  counter
+});
+
+export default rootReducer;
```

Comment:
Any reason not to do directly?

~~~javascript
export default combineReducers({
  counter
});
~~~

(same for todomvc)